mqtt-client-Elixir/mqtt.exs

```diff
@@ -0,0 +1,53 @@
+Mix.install([
+  {:emqtt, github: "emqx/emqtt", tag: "1.8.2"}
+])
+
+require Logger
+
+Process.flag(:trap_exit, true)
+
+topic = "topic-#{:rand.uniform(9999999)}-#{:erlang.system_time(:millisecond)}"
+
+{:ok, subscriber} = :emqtt.start_link(
+  ## Note single quotes
+  host: 'broker.emqx.io',
+  port: 1883,
+  client_id: "elixir_subscriber",
+  username: "user",
+  password: "password"
+)
+{:ok, _} = :emqtt.connect(subscriber)
+
+qos1_granted = 1
+{:ok, %{} = _props, [^qos1_granted]} = :emqtt.subscribe(subscriber, topic, :qos1)
+
+{:ok, publisher} = :emqtt.start_link(
+  ## Note single quotes
```

Comment:
emqtt should really support binary string as hostnames.

Reply:
https://github.com/emqx/emqtt/pull/187